pages/linux/pacman-r.md

```diff
@@ -0,0 +1,7 @@
+# pacman -R
+
+> This command is an alias of `pacman --remove`.
+
+- View documentation for the original command:
+
+`tldr pacman-remove`
```

Comment:
```suggestion
`tldr pacman remove`
```

Reply:
We did `tldr pacman-sync` with the previous page.

Reply:
I've already changed this in #14824.